diff -ur yacas-1.0.47/manualmaker/Makefile.am yacas-1.0.47.new/manualmaker/Makefile.am
--- yacas-1.0.47/manualmaker/Makefile.am	Thu Nov 29 23:43:40 2001
+++ yacas-1.0.47.new/manualmaker/Makefile.am	Thu Nov 29 23:37:49 2001
@@ -14,7 +14,7 @@
 
 yacas_helpdir = $(datadir)/yacas/documentation
 
-books.html: in/arith.chapt in/built.chapt in/calc.chapt \
+books.html$(EXEEXT): in/arith.chapt in/built.chapt in/calc.chapt \
 	in/coding.book in/const.chapt in/functional.chapt  \
 	in/intro.book in/io.chapt in/linalg.chapt in/plugins.chapt \
 	in/lists.chapt in/preds.chapt in/prog.chapt in/controlflow.chapt \
