diff -Nru yacas-1.0.54.orig/src/Makefile.am yacas-1.0.54/src/Makefile.am
--- yacas-1.0.54.orig/src/Makefile.am	Sun Jan 12 23:42:48 2003
+++ yacas-1.0.54/src/Makefile.am	Mon Apr  7 12:33:37 2003
@@ -114,25 +114,25 @@
 	test -z "yacasdebug" || rm -rf yacasdebug
 
 install-data-local: yacas.h
-	$(mkinstalldirs) $(yacas_include)/
-	$(mkinstalldirs) $(yacas_lib)/
-	$(mkinstalldirs) $(yacas_include)/plat/
-	$(mkinstalldirs) $(yacas_include)/plat/linux32/
+	$(mkinstalldirs) $(DESTDIR)$(yacas_include)/
+	$(mkinstalldirs) $(DESTDIR)$(yacas_lib)/
+	$(mkinstalldirs) $(DESTDIR)$(yacas_include)/plat/
+	$(mkinstalldirs) $(DESTDIR)$(yacas_include)/plat/linux32/
 	-for file in $(srcdir)/*.h; do \
 	basefile=`basename $$file`; \
-	$(INSTALL_DATA) $$file $(yacas_include)/$$basefile; \
+	$(INSTALL_DATA) $$file $(DESTDIR)$(yacas_include)/$$basefile; \
 	done
 	-for file in $(srcdir)/*.inl; do \
 	basefile=`basename $$file`; \
-	$(INSTALL_DATA) $$file $(yacas_include)/$$basefile; \
+	$(INSTALL_DATA) $$file $(DESTDIR)$(yacas_include)/$$basefile; \
 	done
 	-for file in $(srcdir)/plat/linux32/*.h; do \
 	basefile=`basename $$file`; \
-	$(INSTALL_DATA) $$file $(yacas_include)/plat/linux32/$$basefile; \
+	$(INSTALL_DATA) $$file $(DESTDIR)$(yacas_include)/plat/linux32/$$basefile; \
 	done
 	-for file in $(srcdir)/plat/linux32/*.inl; do \
 	basefile=`basename $$file`; \
-	$(INSTALL_DATA) $$file $(yacas_include)/plat/linux32/$$basefile; \
+	$(INSTALL_DATA) $$file $(DESTDIR)$(yacas_include)/plat/linux32/$$basefile; \
 	done
 
 version.h: ../config.h
