diff -Nru yacas-1.0.43/Makefile.am yacas-1.0.43.new/Makefile.am
--- yacas-1.0.43/Makefile.am	Mon Nov 13 06:20:25 2000
+++ yacas-1.0.43.new/Makefile.am	Sun Dec 10 13:31:37 2000
@@ -3,7 +3,8 @@
 EXTRA_DIST = yacas.spec yacas.spec.in makemake mkhomepage mkmanual yacas.sh yacas_client doxy_yacas yacas.el
 
 install-data-local:
-	-cp yacas_client $(bindir)/
+	$(mkinstalldirs) $(DESTDIR)$(bindir)
+	$(INSTALL) yacas_client $(DESTDIR)$(bindir)
 
 test:
 	cd scripts ; ../src/yacas examples/tests
diff -Nru yacas-1.0.43/docs/Makefile.am yacas-1.0.43.new/docs/Makefile.am
--- yacas-1.0.43/docs/Makefile.am	Thu Nov  9 20:22:51 2000
+++ yacas-1.0.43.new/docs/Makefile.am	Sun Dec 10 13:29:08 2000
@@ -9,8 +9,7 @@
 
 
 install-data-local:
-	-mkdir $(scriptdir)
-	-mkdir $(scriptdir)documentation/
-	-cp yacaslogo.gif $(scriptdir)documentation/
+	$(mkinstalldirs) $(DESTDIR)$(scriptdir)/documentation/
+	$(INSTALL_DATA) yacaslogo.gif $(DESTDIR)$(scriptdir)/documentation/
 
 
diff -Nru yacas-1.0.43/manualmaker/Makefile.am yacas-1.0.43.new/manualmaker/Makefile.am
--- yacas-1.0.43/manualmaker/Makefile.am	Thu Nov  9 20:22:52 2000
+++ yacas-1.0.43.new/manualmaker/Makefile.am	Sun Dec 10 13:27:35 2000
@@ -48,10 +48,10 @@
 	-cp $(srcdir)/*.html  $(distdir) 
 
 install-data-local: books.html 
-	$(mkinstalldirs) $(yacas_helpdir)/
+	$(mkinstalldirs) $(DESTDIR)$(yacas_helpdir)/
 	-for file in $(srcdir)/*.html; do \
 	basefile=`basename $$file`; \
-	$(INSTALL_DATA) $(srcdir)/$$file $(yacas_helpdir)/$$basefile; \
+	$(INSTALL_DATA) $(srcdir)/$$file $(DESTDIR)$(yacas_helpdir)/$$basefile; \
 	done
 
 
diff -Nru yacas-1.0.43/scripts/Makefile.am yacas-1.0.43.new/scripts/Makefile.am
--- yacas-1.0.43/scripts/Makefile.am	Thu Nov  9 20:22:54 2000
+++ yacas-1.0.43.new/scripts/Makefile.am	Sun Dec 10 13:30:29 2000
@@ -14,10 +14,10 @@
 script_DATA = $(SCRIPTFILES)
 
 install-data-local:
-	-mkdir $(scriptdir)addons/
-	-cp addons/README $(scriptdir)addons/
-	-cp addons/editvi* $(scriptdir)addons/
-	-cp addons/gnuplot* $(scriptdir)addons/
+	$(mkinstalldirs) $(DESTDIR)$(scriptdir)/addons/
+	$(INSTALL_DATA) addons/README $(DESTDIR)$(scriptdir)/addons/
+	$(INSTALL_DATA) addons/editvi* $(DESTDIR)$(scriptdir)/addons/
+	$(INSTALL_DATA) addons/gnuplot* $(DESTDIR)$(scriptdir)/addons/
 
 EXTRA_DIST = $(SCRIPTFILES) \
 	examples/tests \
